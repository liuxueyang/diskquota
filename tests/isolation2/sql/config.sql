--start_ignore
CREATE DATABASE diskquota;
--end_ignore

!\retcode gpconfig -c shared_preload_libraries -v diskquota;
!\retcode gpstop -raf;

!\retcode gpconfig -c diskquota.naptime -v 0;
!\retcode gpconfig -c max_worker_processes -v 20;
!\retcode gpstop -raf;

-- Show the values of all GUC variables
1: SHOW diskquota.naptime;
1: SHOW diskquota.max_active_tables;
1: SHOW diskquota.worker_timeout;
 