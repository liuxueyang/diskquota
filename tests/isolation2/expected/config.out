
!\retcode gpconfig -c shared_preload_libraries -v diskquota;
(exited with code 0)
!\retcode gpstop -raf;
(exited with code 0)

!\retcode gpconfig -c diskquota.naptime -v 0;
(exited with code 0)
!\retcode gpconfig -c max_worker_processes -v 20;
(exited with code 0)
!\retcode gpstop -raf;
(exited with code 0)

-- Show the values of all GUC variables
1: SHOW diskquota.naptime;
 diskquota.naptime 
-------------------
 0                 
(1 row)
1: SHOW diskquota.max_active_tables;
 diskquota.max_active_tables 
-----------------------------
 1048576                     
(1 row)
1: SHOW diskquota.worker_timeout;
 diskquota.worker_timeout 
--------------------------
 60                       
(1 row)

